pkg/proxy/nftables/proxier_test.go

```diff
@@ -366,6 +373,7 @@ func TestOverallNFTablesRules(t *testing.T) {
 		add element ip kube-proxy no-endpoint-nodeports { tcp . 3001 comment "ns2/svc2:p80" : drop }
 		add element ip kube-proxy no-endpoint-services { 1.2.3.4 . tcp . 80 comment "ns2/svc2:p80" : drop }
 		add element ip kube-proxy no-endpoint-services { 192.168.99.22 . tcp . 80 comment "ns2/svc2:p80" : drop }
+		add element ip kube-proxy nodeport-ips { 192.168.0.2 }
```

Comment:
this doesn't really go with `# svc2`; it should go earlier in the dump. You could put it right after the `add chain nodeport-ips` line